packages/tools/pluggable-widgets-tools/src/typings-generator/__tests__/outputs/association.ts

```diff
@@ -19,7 +19,11 @@ export interface MyWidgetContainerProps {
 }
 
 export interface MyWidgetPreviewProps {
+    /**
+     * @deprecated since version 9.18.0. Please use class property instead.
```

Comment:
Most probably this will look in an IDE like: `since version 9.18.0. Please use class property instead`.  The word `deprecated` is not the part of the sentence an IDE will show. Please adjust the text accordingly.